lld/test/MachO/objc-relative-method-lists-default-off-on-mac10.s

```diff
@@ -0,0 +1,244 @@
+# REQUIRES: aarch64
```

Comment:
This test seems copied from `objc-relative-method-lists-simple.s`. Is there a reason to copy the whole thing - rather than doing it directly in `objc-relative-method-lists-simple.s` ?

Reply:
They are slightly different and needs to be highlighted (minos 10 vs 11, output difference). Do you prefer to have them merged?

Reply:
Yes, better to have them merged. 
If you're referring to the existing `.build_version macos, 11, 0` In `objc-relative-method-lists-simple.s` - I think it just works removing that and having the command line flags be the determining one.